### PR TITLE
GH-1091: Moved selected repository to the status bar.

### LINKED ISSUE #1091: [git] Move 'current' repository to status bar
We should move the display of the currently selected git repository to the status bar and have a general command to change it using the quick command.

The name of the repo should be shortened to max 8 characters and be displays in front of the branch, only if there are other repos in the current workspace. Clicking the name shall trigger the 'Git: Change Repostory' command.

### PR DESCRIPTION
**Pull request by kittaakos**
Closes #1091.

### CODE REVIEW

File: packages/git/src/browser/git-command.ts [hunk 1]
```diff
@@ -31,6 +31,10 @@ export namespace GIT_COMMANDS {
         id: 'git.checkout',
         label: 'Git Checkout'
     };
+    export const CHANGE_REPOSITORY = {
+        id: 'git.change.repository',
+        label: 'Git Change Repository'
```
[the remark this answers is not in the document]
Comment (svenefftinge) [reply packages/git/src/browser/git-command.ts]: Created https://github.com/theia-ide/theia/issues/1119

**Comment by kittaakos** [comment]
> Please show the full name.

👍

**Comment by kittaakos** [comment]
One more thing on this PR: since we disable the Git Change Repository command (and remove the status bar contribution too) when there is only one (or zero) repository, I think, we have to remove the currently selected one from the list of proposed quick open items. Or at least, we have to indicate this in the quick open item description.

So if we know repository A, B, and C. And C is the selected one, we need to propose only A and B but not C when switching between the repositories. Currently, this is not covered by this PR.

Any feedbacks on this?

**Comment by svenefftinge** [comment]
For change theme I show all themes. I think it's good; as you get an overview of what is available.

**Comment by kittaakos** [comment]
> For change theme I show all themes. I think it's good; as you get an overview of what is available.

OK, then I leave the current functionally as is.

**Comment by kittaakos** [comment]
I have applied the changes based on the review feedbacks. We have the green build; I am merging this PR.

**Pull request merged**